### ISSUE TITLE
Project Discontinued Again

### ISSUE THREAD

**Issue by itsmerosu** [mahtab2003/Xera#190]
Is the Xera project discontinued again? I check for new updates daily, but nothing has changed.

**Comment by NotKRhero** [comment]
i think there are busy but u can try to update urself too
i updated a lil bit dash.alwaysfree.us.kg look

**Comment by itsmerosu** [comment]
> i think there are busy but u can try to update urself too i updated a lil bit dash.alwaysfree.us.kg look

Checked, but it lacks interesting features. only has some labels and social media options.

**Comment by NotKRhero** [comment]
yeah ;< but i have a nice theme also it changed the all icons bruh

**Comment by itsmerosu** [comment]
> yeah ;< but i have a nice theme also it changed the all icons bruh

I checked, but not too thoroughly.

**Comment by NotKRhero** [comment]
huh how can u check it without i choose the theme ;\

**Comment by itsmerosu** [comment]
> huh how can u check it without i choose the theme ;\

🤣

**Comment by NotKRhero** [comment]
check again and i am going to create a pull req i added undo redo to ticket system bruh

**Comment by NotKRhero** [comment]
i added more things

**Comment by NotKRhero** [comment]
and you can change the nav bar as a side bar if you want from changing the code a lil bit

**Comment by NotKRhero** [comment]
added team section too

**Comment by itsmerosu** [comment]
> check again and i am going to create a pull req i added undo redo to ticket system bruh



> i added more things



> and you can change the nav bar as a side bar if you want from changing the code a lil bit



> added team section too

Please pull it on dev repo

**Comment by NotKRhero** [comment]
https://alwaysfree.us.kg/delete1jns.png check this out

**Comment by itsmerosu** [comment]
> https://alwaysfree.us.kg/delete1jns.png check this out

Nice UI

**Comment by bixacloud** [comment]
> > https://alwaysfree.us.kg/delete1jns.png check this out
> 
> Nice UI

it is bixa my project

**Comment by NotKRhero** [comment]
@itsmerosu get bixa now its so cool man

**Comment by alex4642** [comment]
dude alwaysfree.us.kg is down? can you give me discord server link, i accidently quit

**Comment by NotKRhero** [comment]
alwaysfree.us.kg moved to alwaysfreehost.com : ) ;/ btw u can join my discord by discord.afhz.org
